script/bootstrap

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+#
+#  Copyright (c) 2017, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+. "$(dirname "$0")"/_initrc
+
+install_packages_apt()
+{
+    sudo apt-get install -y build-essential
+    # For wpantund
+    sudo apt-get install -y libdbus-1-dev
+
+    # This is for configuring wpantund
+    sudo apt-get install -y autoconf-archive
+
+    # mDNS
+    sudo apt-get install -y libavahi-core-dev
+
+    # Doxygen
+    sudo apt-get install -y doxygen
+
+    # For libcoap to generate .sym and .map
+    sudo apt-get install -y ctags
+
+    # Boost
+    sudo apt-get install -y libboost-dev libboost-filesystem-dev libboost-system-dev
+
+    # npm bower
+    if ! which bower; then
+        curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
```

Comment:
Thanks, added curl. And downgraded to nodejs 6.x, which is the stable version and is more widely supported.